watertap/unit_models/tests/test_osmotically_assisted_reverse_osmosis_0D.py

```diff
@@ -465,6 +484,237 @@ def test_solution(self, RO_frame):
         assert pytest.approx(0, abs=1e-3) == value(m.fs.unit.feed_side.deltaP[0])
         assert pytest.approx(0, abs=1e-3) == value(m.fs.unit.permeate_side.deltaP[0])
 
+    @pytest.fixture(scope="class")
+    def RO_SKK_frame(self):
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(dynamic=False)
+
+        m.fs.properties = props.NaClParameterBlock()
+
+        m.fs.unit = OsmoticallyAssistedReverseOsmosis0D(
+            property_package=m.fs.properties,
+            has_pressure_change=True,
+            concentration_polarization_type=ConcentrationPolarizationType.fixed,
+            mass_transfer_coefficient=MassTransferCoefficient.none,
+            transport_model=TransportModel.SKK,
+            has_full_reporting=True,
+        )
+
```

Comment:
I think it will make more sense if we have the tets for solving the system with `concentration_polarization_type` and `mass_transfer_coefficient` set as `calculated` because it is the most general case.